pkg/diagnose/runner.go

```diff
@@ -43,6 +43,7 @@ func RunAll(w io.Writer) error {
 		statusString := color.GreenString("PASS")
 		if err != nil {
 			statusString = color.RedString("FAIL")
+			log.Info(err)
```

Comment:
I'm fine with this, but wondering if we should maybe be a bit explicit in case the returned error isn't particularly well worded:
```
log.Infof("diagnosis error for %s: %v", name, err)
```

Reply:
Good point.  I'll make that change and merge.

Reply:
..if it's green